From: Aurelien Jarno <aurelien@aurel32.net>
Date: Sun, 03 Dec 2017 13:17:05 +0100
Subject: s390/qeth: Avoid ABI change in 4.9.62
Forwarded: not-needed

Commit 1034051045d1 's390/qeth: issue STARTLAN as first IPA command'
changed qeth_send_startlan to static and stopped exporting it.
Following the changes needed to issue STARTLAN as first IPA command,
it is indeed used in single place.

This patch partially reverts the change so that qeth_send_startlan is
kept being exported.

---
--- a/drivers/s390/net/qeth_core.h
+++ b/drivers/s390/net/qeth_core.h
@@ -921,6 +921,7 @@
 int qeth_core_hardsetup_card(struct qeth_card *);
 void qeth_print_status_message(struct qeth_card *);
 int qeth_init_qdio_queues(struct qeth_card *);
+int qeth_send_startlan(struct qeth_card *);
 int qeth_send_ipa_cmd(struct qeth_card *, struct qeth_cmd_buffer *,
 		  int (*reply_cb)
 		  (struct qeth_card *, struct qeth_reply *, unsigned long),
--- a/drivers/s390/net/qeth_core_main.c
+++ b/drivers/s390/net/qeth_core_main.c
@@ -2944,7 +2944,7 @@
 }
 EXPORT_SYMBOL_GPL(qeth_send_ipa_cmd);
 
-static int qeth_send_startlan(struct qeth_card *card)
+int qeth_send_startlan(struct qeth_card *card)
 {
 	int rc;
 	struct qeth_cmd_buffer *iob;
@@ -2957,6 +2957,7 @@
 	rc = qeth_send_ipa_cmd(card, iob, NULL, NULL);
 	return rc;
 }
+EXPORT_SYMBOL_GPL(qeth_send_startlan);
 
 static int qeth_default_setadapterparms_cb(struct qeth_card *card,
 		struct qeth_reply *reply, unsigned long data)
